modules/cnf-scheduling-numa-aware-workloads.adoc

```diff
@@ -0,0 +1,127 @@
+
+.Procedure
+
+. Get the name of the NUMA-aware scheduler that is deployed in the cluster:
++
+[source,terminal]
+----
+$ oc get numaresourcesschedulers.nodetopology.openshift.io numaresourcesscheduler -o json | jq '.status.schedulerName'
+----
++
+.Example output
+[source,terminal]
+----
+topo-aware-scheduler
+----
+
+. Create a `Deployment` CR that uses `topo-aware-scheduler`, for example:
+
+.. Save the following YAML in the `nro-deployment.yaml` file:
++
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: numa-deployment-1
+  namespace: openshift-numaresources
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: test
+  template:
+    metadata:
+      labels:
+        app: test
+    spec:
+      schedulerName: topo-aware-scheduler
+      containers:
+      - name: ctnr
+        image: quay.io/openshifttest/hello-openshift:openshift
+        imagePullPolicy: IfNotPresent
+        resources:
+          limits:
+            memory: "100Mi"
+            cpu: "10"
+          requests:
+            memory: "100Mi"
+            cpu: "10"
+      - name: ctnr2
+        image: gcr.io/google_containers/pause-amd64:3.0
+        imagePullPolicy: IfNotPresent
+        command: ["/bin/sh", "-c"]
+        args: [ "while true; do numalign; sleep 100000; done;" ]
+        resources:
+          limits:
+            memory: "100Mi"
+            cpu: "8"
+          requests:
+            memory: "100Mi"
+            cpu: "8"
+----
+
+.. Create the `Deployment` CR:
++
+[source,terminal]
+----
+$ oc create -f nro-deployment.yaml
+----
+
+.Verification
+
+. Verify that the deployment was successful:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-numaresources
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                                READY   STATUS    RESTARTS   AGE
+numa-deployment-1-56954b7b46-pfgw8                  2/2     Running   0          129m
+numaresources-controller-manager-7575848485-bns4s   1/1     Running   0          15h
+numaresourcesoperator-worker-dvj4n                  2/2     Running   0          18h
+secondary-scheduler-56994cf6cf-7qf4q                1/1     Running   0          18h
+----
+
+. Verify that the `topo-secondary-scheduler` is scheduling the deployed pod:
++
+[source,terminal]
+----
+$ oc describe pod numa-deployment-1-56954b7b46-pfgw8 -n openshift-numaresources
+----
++
+.Example output
+[source,terminal]
+----
+Events:
+  Type    Reason          Age   From                  Message
+  ----    ------          ----  ----                  -------
+  Normal  Scheduled       130m  topo-aware-scheduler  Successfully assigned openshift-numaresources/numa-deployment-1-56954b7b46-pfgw8 to compute-0.example.com
+----
+
+[NOTE]
+====
+Deployments that request more resources than is available for scheduling will fail with a `MinimumReplicasUnavailable` error. The deployment will succeed when the required resources become available.
```

Comment:
+ hence, the pods will keep on pending until the required resources become available.